wordpress-mobile/WordPress-iOS
# Aztec: Adds analytic events for all the new toolbar buttons/actions

Closes #7408 

Adds analytic events to all of the button presses for the new toolbar UI. Aztec previously had no analytics tracking on the toolbar so these are all new events for the view controller.

To test:
1. Open `TracksServiceRemote.m` (from a pod) and edit `sendBatchOfEvents:` to dump `events` out with NSLog.
2. Make sure Aztec is your editor.
3. Tap each button at least once.
4. When selecting Header also make sure to select a style.
5. Go to Tracks (or view the log dumps from step 1) and verify events were sent through.

Needs review: @frosty 

cc: @theck13
